pytket/tests/mapping_test.py

```diff
@@ -209,8 +216,64 @@ def test_basic_mapping() -> None:
     assert len(circ.get_commands()) == 10
 
 
+def test_MultiGateReorderRoutingMethod() -> None:
```

Comment:
Could you please add a testcase to check the interaction with with the lexi placement?